modules/jaeger-deploy-default.adoc

```diff
@@ -108,3 +108,8 @@ Once the installation process has completed, you should see output similar to th
 NAME                                         READY   STATUS    RESTARTS   AGE
 jaeger-all-in-one-inmemory-cdff7897b-qhfdx   2/2     Running   0          24s
 ----
+
+[NOTE]
```

Comment:
You don't actually need this note, this module does not deploy the streaming strategy.

Reply:
If you look at the preview you can see the flow here.  This note appears at the end of the topic for how to deploy the default Jaeger configuration.  It is not needed in this section.

https://deploy-preview-28313--osdocs.netlify.app/openshift-enterprise/latest/jaeger/jaeger_install/rhbjaeger-deploying.html#jaeger-deploy-default_jaeger-deploying